maintainers/maintainer-list.nix

```diff
@@ -9848,6 +9848,12 @@
     githubId = 6498458;
     name = "pebble kite";
   };
+  pbsds = {
```

Comment:
I see you're adding yourself to the maintainers list, do you also want to add yourself as a maintainer of the librealsense package? I certainly would appreciate others being able to test changes! :)

Reply:
Whoopsy, forgot that step